bedrock/mozorg/views.py

```diff
@@ -422,7 +422,7 @@ def new_home(request, locale=None):
 
 def home(request):
```

Comment:
@jpetto Thanks! I wasn't too sure about this. I will remove HomeTestView and just reference one view.